### DISCUSSION TITLE
about video to video

### DISCUSSION THREAD

**Discussion by Li1177** [continue-revolution/sd-webui-animatediff#346] [Q&A]
I would like to ask how to achieve a certain effect using AnimatDiff for video generation. For example, if the original video is 30 FPS with 600 frames, I want AnimatDiff to generate a video for me with 160 frames at 8 FPS. Then, using the Deform plugin, I would like to insert 3 frames between each of these 160 frames. How should I set this up?

**Comment by CesarCruzUtec** [comment]
maybe convert the source video to the config you want, and use it as the input and do the next steps that you mentioned
